src/components/ha-sidebar.ts

```diff
@@ -658,36 +659,36 @@ class HaSidebar extends SubscribeMixin(LitElement) {
   }
 
   private async _activateEditMode() {
-    if (!Sortable) {
-      const [sortableImport, sortStylesImport] = await Promise.all([
-        import("sortablejs/modular/sortable.core.esm"),
-        import("../resources/ha-sortable-style"),
-      ]);
-
-      const style = document.createElement("style");
-      style.innerHTML = (sortStylesImport.sortableStyles as CSSResult).cssText;
-      this.shadowRoot!.appendChild(style);
-
-      Sortable = sortableImport.Sortable;
-      Sortable.mount(sortableImport.OnSpill);
-      Sortable.mount(sortableImport.AutoScroll());
-    }
+    await Promise.all([this._loadSortableStyle(), this._createSortable()]);
+  }
 
-    await this.updateComplete;
+  private async _loadSortableStyle() {
+    if (this.sortableStyleLoaded) return;
+
+    const sortStylesImport = await import("../resources/ha-sortable-style");
 
-    this._createSortable();
+    const style = document.createElement("style");
+    style.innerHTML = (sortStylesImport.sortableStyles as CSSResult).cssText;
+    this.shadowRoot!.appendChild(style);
+
+    this.sortableStyleLoaded = true;
+    await this.updateComplete;
```

Comment:
I'm not sure the style must be lazy loaded because it's not a big file.

Reply:
Lazy is very important here because sidebar is part of the initial load.

Reply:
Run `script/size_stats` to see bundle sizes.

Reply:
> Lazy is very important here because sidebar is part of the initial load.

It isn't actually, the sidebar itself is dynamically loaded

Reply:
It's still loaded as soon as you open the page.